WooCommerce/Classes/ViewRelated/Products/Edit Product/BottomSheetListSelector/ProductTypeBottomSheetListSelectorCommand.swift

```diff
@@ -62,6 +68,12 @@ public enum BottomSheetProductType: Hashable {
                                      comment: "Action sheet option when the user wants to change the Product type to external product")
         case .custom(let title):
             return title
+        case .subscription:
+            return NSLocalizedString("Simple subscription product",
+                                     comment: "Action sheet option when the user wants to change the Product type to Subscription product")
+        case .variableSubscription:
+            return NSLocalizedString("Variable subscription product",
+                                     comment: "Action sheet option when the user wants to change the Product type to Variable subscription product")
```

Comment:
New NSLocalizedStrings should use a reverse domain type key
```suggestion
        case .subscription:
            return NSLocalizedString("product.sheet.option.simpleSubscription.title",
                                     value: "Simple subscription product",
                                     comment: "Action sheet option when the user wants to change the Product type to Subscription product")
        case .variableSubscription:
            return NSLocalizedString("product.sheet.option.variableSubscription.title",
                                     value: "Variable subscription product",
                                     comment: "Action sheet option when the user wants to change the Product type to Variable subscription product")
```

See p91TBi-aJl-p2 for context

Reply:
Good catch. Fixed in https://github.com/woocommerce/woocommerce-ios/pull/11118/commits/21ec2e8f13308e9780e4533184e3e08a390831ce